k8s.gcr.io/lib.sh

```diff
@@ -229,7 +234,7 @@ function empower_promoter() {
     if ! gcloud --project "${project}" iam service-accounts describe "${acct}" >/dev/null 2>&1; then
         gcloud --project "${project}" \
             iam service-accounts create \
-            "${name}" \
+            "${PROMOTER_SVCACCT}" \
```

Comment:
It's not obvious to me why this was changed...

Reply:
This was a bug - name is not defined here.  We never hit it before because testing always stopped at the `if` and never executed this line until we had a legit new project.